vnext/Microsoft.ReactNative.WindowsAppSDK/nuget/nuget_pack.cmd

```diff
@@ -0,0 +1 @@
+nuget pack .\Microsoft.ReactNative.WindowsAppSDK\nuget\Microsoft.ReactNative.WindowsAppSDK.local.nuspec -version 0.0.4 -properties baseplatform=x64;baseconfiguration=Release  
```

Comment:
Is this script meant to be checked in?
If so, consider:

- Using PowerShell (`ps1`) instead.
- Parameterize platform, configuration and version.

Reply:
this file is just for local testing, doesn't need to be checked in (but it was convenient to put it in the branch so agne had it too :) )